File: src/main/service/LoginService.java
package main.service;

import main.dao.LoginDAO;
import main.model.Login;


public class LoginService {

    private LoginDAO loginDAO;

    public LoginService() {
        this.loginDAO = new LoginDAO();
    }

    public String login (String username, String password) {
        return this.loginDAO.login(username, password);
    }

    public boolean InsertLogin (Login login) {
        return this.loginDAO.insertLogin(login);
    }
}

File: contraderOBDAngular/src/main/webapp/js/service/loginService.js
app.service('loginService', function ($http) {
    return {
        user : {},

        login: function (params, callback) {
            $http({
                method: 'POST',
                url: '/login',
                params: params
            }).then(function successCallback(response) {
                if (callback) {
                    callback(response);
                }
            }, function errorCallback(response) {
            });
        },
        init: function (user) {
            let parent = this;
            this.user = user;
        },
        getUsername: function () {
            return this.user.username;
        },
        getRuolo: function () {
            return this.user.ruolo;
        },
        getId: function () {
            return this.user.id;
        },
        logout: function () {
            this.user = {};
        }
    };
})